libs/shared/ui/src/lib/components/sticky-action-form-toaster/sticky-action-form-toaster.tsx

```diff
@@ -0,0 +1,49 @@
+import Button, { ButtonStyle } from '../buttons/button/button'
+
+export interface StickyActionFormToasterProps {
+  description?: string
+  onReset?: () => void
+  resetLabel?: string
+  onSubmit?: () => void
+  submitLabel?: string
+  className?: string
+  disabledValidation?: boolean
+}
+
+export function StickyActionFormToaster(props: StickyActionFormToasterProps) {
+  const {
+    description = 'Warning, there are still unsaved changes!',
+    onReset,
+    resetLabel = 'Reset',
+    submitLabel = 'Save modifications',
+    onSubmit,
+    className = '',
+  } = props
+  return (
+    <div
+      className={`rounded bg-element-light-darker-100 shadow-xl text-white ${className} inline-flex items-center pl-4 p-2 gap-10`}
+    >
+      {description && <span className="font-medium text-white text-sm">{description}</span>}
+      <div className="flex gap-5">
+        {resetLabel && onReset && (
+          <button type="button" className="text-ssm underline font-medium" onClick={() => onReset()}>
+            {resetLabel}
+          </button>
+        )}
+        {submitLabel && onSubmit && (
+          <Button
+            style={ButtonStyle.CONFIRM}
+            className=""
+            onClick={() => onSubmit()}
+            disabled={props.disabledValidation}
+            dataTestId={'submit-button'}
```

Comment:
can be directly  `="submit-button"`

Reply:
Yes Intellij is suppose to give me the good {} or '' in function of the type of the property but I don't know why for him dataTestId can be other than a string so it open me these {} automatically... What a douchebag